src/Features/Core/Portable/StackTraceExplorer/DotnetStackFrameParser.cs

```diff
@@ -0,0 +1,59 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Diagnostics.CodeAnalysis;
+using Microsoft.CodeAnalysis.Text;
+
+namespace Microsoft.CodeAnalysis.StackTraceExplorer
+{
+    internal sealed class DotnetStackFrameParser : IStackFrameParser
```

Comment:
i like that the parsing is abstracted out.  do yo uhave specific unit tests just for the parser.  If not, can you do that?

Reply:
These are currently tested thoroughly in the StackTraceExplorer tests. Do you think it's worth separating further?